package.json

```diff
@@ -18,6 +18,18 @@
       "title": "Works for Private Browsing",
       "type": "bool",
       "value": true
+    },
+    {
+      "name": "maxOpenInNextWindow",
+      "title": "Open in next available window",
```

Comment:
Perhaps make this more descriptive, like `Open in next available window if exceeded maximum number of tabs`.